packages/core/src/oidc/init.ts

```diff
@@ -92,9 +92,8 @@ export default function initOidc(queries: Queries): Provider {
             throw new errors.InvalidTarget();
```

Comment:
> fix getResourceServerInfo userId: use ctx.session.userId as the userId, because the previous one may not be empty.

may not be empty or empty? I'm wondering the difference between `oidc.session?.accountId` and `oidc.account.accountId`

Reply:
There are so many cases that this function be called, `oidc.account.accountId` is only valid when the user has completed with the token granting flow.

Reply:
isee